### PR TITLE
Create pomegranate-lemon-mint

### PR DESCRIPTION
**Pull request by bookish**

### CODE REVIEW

File: pomegranate-lemon-mint [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ],
+    "directions": [
+        "<Pour ice cubes into a large glass pitcher.>",
+        "<Slice the pomegranate in half, then squeeze the juice from both halves over the ice cubes in the pitcher.  It is okay if a few of the seeds end up in the pitcher because they are edible.>",
+        "<Slice the lemon in half and squeeze the lemon juice into the pitcher.>"
+        "<Pour the sparkling water into the pitcher.>"
+        "<Wash the mint and put the nicest leaves into the pitcher.>"
+        "<Pour in a tall glass and enjoy!>"
+    ],
+    "image": "<drink-name.jpg>",
```
Comment (alfg) [inline pomegranate-lemon-mint]: Add drink name here.

File: pomegranate-lemon-mint [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "source": "<http://cheekeelee.blogspot.com/2012/03/jamie-olivers-pomegranate-soda.html>",
+    "keywords": [
+        "<Jamie Oliver sparkling water pomegranate mint lemon ice non-alcoholic fruit>",
```
Comment (alfg) [inline pomegranate-lemon-mint]: These should be something like:

```suggestion
        "keywords": ["sparking water", "pomegranate", "mint", "lemon", "ice", "non-alcoholic", "fruit"]
```

File: pomegranate-lemon-mint [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        "...",
```
Comment (alfg) [inline pomegranate-lemon-mint]: Remove.
```suggestion
```

File: pomegranate-lemon-mint [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "<Pomegranate Lemon Mint (adapted from Jamie Oliver the wonderful chef>",
+    "description": "<Refreshing>",
+    "github": "<bookish>",
+    "ingredients": [
+        "<One litre bottle of sparkling water>",
+        "<One whole pomegranate>",
+        "<One bunch of fresh mint>"
+        "<One whole lemon>"
+         "<large ice cubes>"
+    ],
```
Comment (alfg) [inline pomegranate-lemon-mint]: See updated ingredients schema at:
https://github.com/alfg/opendrinks/blob/master/CONTRIBUTING.md

File: pomegranate-lemon-mint (hunk 1 above)
Comment (bookish) [reply pomegranate-lemon-mint]: Pomegranate Lemon Mint Refresher

**Comment by alfg** [comment]
Closing. Feel free to re-open the PR with the suggested fixes.

**Pull request closed without merging**